specification/securityandcompliance/resource-manager/readme.go.md

```diff
@@ -23,4 +23,18 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag)=='package-2021-01-11' && $(go)
 output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-01-11/$(namespace)
+```
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2021-03-08
+```
```

Comment:
Could we remove this and add this entry to the batch section above on line 15?

Reply:
Do you mean like this?:
``` yaml $(go) && $(multiapi)
batch:
  - tag: package-2021-01-11
  - tag: package-2021-03-08
```
Check failed after I change it to this.

Reply:
That should be irrelevant. Let us try this first

Reply:
I do not really understand it, can you be more specific?

Reply:
but it will throw exceptions if I move this entry up, Updated readme.md file · Azure/azure-rest-api-specs@4da7043 (github.com)